e2e/cypress/integration/plugins/marketplace/marketplace_spec.js

```diff
@@ -310,8 +310,8 @@ describe('Plugin Marketplace', () => {
                 cy.get('#marketplaceTabs-pane-installed').should('be.visible');
                 cy.get('#marketplaceTabs-pane-allPlugins').should('not.exist');
 
-                // # click on Install Plugins should change current tab
-                cy.findByText('Install Plugins').should('be.visible').click();
+                // # click on "All Plugins" should change current tab
+                cy.findByText('All Plugins').should('be.visible').click();
```

Comment:
This test was broken. @lieut-data Could you please double check that I fixed the test in the right way. https://github.com/mattermost/mattermost-webapp/commit/54737c9db4d4b0fafe21567479b22cb6b2bd45a7#diff-91137a34d5c4d9c156f392f16f2bd819 was the last commit that changed this.

Reply:
@hanzei I believe this PR fixed it: https://github.com/mattermost/mattermost-webapp/pull/4613/files#diff-91137a34d5c4d9c156f392f16f2bd819

Reply:
Ah, got it. Thanks for the explanation.

The test did fail for me, because I had a plugin previously installed. Filled https://mattermost.atlassian.net/browse/MM-21610 to track this and reverted my "fix".

Reply:
I had included in this PR https://github.com/mattermost/mattermost-webapp/pull/4564/files#diff-91137a34d5c4d9c156f392f16f2bd819 a change to always clean up the installed plugins after the tests so maybe that's why I wasn't getting that problem.

Reply:
Still, the test will fail, if any existing plugins are present. To me the ticket is valid.